CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+<!-- SPDX-FileCopyrightText: The Ferrocene Developers -->
+<!-- SPDX-License-Identifier: MIT OR Apache-2.0 -->
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [1.0.1] - 2024-05-29
+
+### Fixed
+
+- Versioning issue where `--version` was still showing `0.0.0`.
```

Comment:
In the example changelog they include PR numbers: https://github.com/olivierlacan/keep-a-changelog/blob/main/CHANGELOG.md#added-1

I like this, can we do that?

Reply:
Ah yes! Let me add that.

Reply:
Maybe we could even link??

Reply:
Oh, wait, does this create a chicken and egg problem? Where you need to open your PR to get your link? Oh no.

Reply:
I think this PR is a special case because it is a versioning snafu AND a release.